### PR TITLE
Allow to set left indent for tabs

### PR DESCRIPTION
**Pull request by denis-bunakalya**
Signed-off-by: Denis Bunakalya <denis.bunakalya@xored.com>

# Contribution checklist

## Checklist

* [ ] - UI test added to added/changed functionality?
* [ ] - Screenshot is added to PR if applicable ?
* [x] - Does the code work? Check whether function and logic are correct.
* [ ] - Does Changelog.md is updated with changes?
* [x] - Does the translations are up to date?
* [x] - Does it well tested?
* [x] - Tested for Chrome.
* [ ] - Tested for Safari.
* [x] - Go through the changed code looking for typos, TODOs, commented LOCs, debugging pieces of code, etc.
* [x] - Rebase your branch onto master and upstream branch
* [x] - Is there any redundant or duplicate code?
* [ ] - Are required links are linked to PR?
* [x] - Does new code is well documented ?

## Related issues

A list of closed updated issues

### CODE REVIEW

File: plugins/contact-resources/src/components/ContactsTabs.svelte [hunk 1]
```diff
@@ -33,9 +33,7 @@
 </script>
 
 {#if tabs && tabs.length > 1}
-  <div class="pl-2">
```
Comment (haiodo) [inline plugins/contact-resources/src/components/ContactsTabs.svelte:36]: I would recommend just to use 'p-2' or 'p-3' class for this div.

File: packages/ui/src/components/Tabs.svelte [hunk 2]
```diff
@@ -20,13 +20,15 @@
 
   export let model: TabModel
   export let selected = 0
+  export let hasLeftIndent = false
```
Comment (haiodo) [inline packages/ui/src/components/Tabs.svelte]: I do not think we need to add this to Tab component.

Comment (denis-bunakalya) [reply packages/ui/src/components/Tabs.svelte]: Removed

File: plugins/contact-resources/src/components/ContactsTabs.svelte (hunk 1 above)
Comment (denis-bunakalya) [reply plugins/contact-resources/src/components/ContactsTabs.svelte:36]: Used padding in Tabs

File: packages/ui/src/components/Tabs.svelte [hunk 3]
```diff
@@ -57,6 +57,7 @@
     flex-shrink: 0;
     flex-wrap: nowrap;
     margin-bottom: 0.5rem;
+    padding: 0 1.5rem;
```
Comment (haiodo) [inline packages/ui/src/components/Tabs.svelte:60]: I not look good in Documents and other places.

image

**Pull request closed without merging**